odkx-src/sync-endpoint-cloud-setup.rst

```diff
@@ -577,16 +577,19 @@ Launching the ODK-X Server
   .. image:: /img/setup-azure/azure11.png
    :width: 600
 
-.. _sync-endpoint-setup-take-stack-down:
+.. _sync-endpoint-setup-take-stack-down-and-bring-back-up:
 
-Take the Stack/Swarm Down
+Take the Stack/Swarm Down and Bringing it Back Up
```

Comment:
In terms of tense, "Take the Stack/Swarm Down and Bring it Back Up" or "Taking the Stack/Swarm Down and Bringing it Back Up"

Reply:
Done